### PR TITLE
UX updates

### PR DESCRIPTION
**Pull request by johnpryan**
this cleans up some smaller UX issues, next step is to build the "new pad" dialog (#1280)

### CODE REVIEW

File: lib/experimental/scss/colors.scss [hunk 1]
```diff
@@ -36,8 +36,8 @@ $dark-secondary-background-color: $dark-code-background-color;
 $dark-gutter-background-color: #121a25;
 $dark-border-color: $dark-gutter-background-color;
 $dark-issues-background-color: lighten($dark-code-background-color, 10%);
-$dark-issue-label-color: $dark-text-color;
-$dark-flash-text-color: #e6e6e6;
+$dark-issue-label-color: #e6e6e6;
+$dark-flash-text-color:$dark-issue-label-color;
```
Comment (RedBrogdon) [inline lib/experimental/scss/colors.scss]: nit: spacing

File: lib/new_playground.dart [hunk 2]
```diff
@@ -65,10 +65,11 @@ class Playground implements GistContainer, GistController {
   MDCButton runButton;
   MDCButton editorConsoleTab;
   MDCButton editorDocsTab;
+  MDCButton closePanelButton;
   DElement editorPanelFooter;
   MDCMenu samplesMenu;
   Dialog dialog;
-  DContentEditable titleEditable;
+  DElement titleEditable;
```
Comment (RedBrogdon) [inline lib/new_playground.dart]: nit: Consider a different name if this isn't user-editable.

File: web/new_playground.html [hunk 3]
```diff
@@ -56,16 +56,28 @@
...
     </div>
     <div class="header-gist-name"></div>
     <div>
         <button type="button" id="share-button" class="mdc-button">Share</button>
     </div>
     <div>
-        <button type="button" id="samples-dropdown-button" class="mdc-button">Samples</button>
+        <button type="button" id="samples-dropdown-button" class="mdc-button">
+            <span class="mdc-button__label">Samples</span>
+            <i class="material-icons mdc-button__icon" aria-hidden="true">expand_more</i>
```
Comment (RedBrogdon) [inline web/new_playground.html]: This is the only aria-* attribute in the PR. Is it necessary for every user, or just an instance of the kind of accessibility metadata we should be putting everywhere?

Comment (johnpryan) [reply web/new_playground.html]: this is coming from the material web components docs. issue created here: https://github.com/dart-lang/dart-pad/issues/1297

File: lib/new_playground.dart (hunk 2 above)
Comment (johnpryan) [reply lib/new_playground.dart]: fixed

File: lib/experimental/scss/colors.scss (hunk 1 above)
Comment (johnpryan) [reply lib/experimental/scss/colors.scss]: fixed

**Pull request merged**